packages/core-transactions/src/handlers/delegate-resignation.ts

```diff
@@ -8,21 +9,59 @@ export class DelegateResignationTransactionHandler extends TransactionHandler {
     }
 
     public async bootstrap(connection: Database.IConnection, walletManager: State.IWalletManager): Promise<void> {
-        return;
+        // XXX should we do something here?
```

Comment:
Yes, here we have to update the wallets and apply the `resignated`  status.
See the other handlers, but basically:
1) Get all resignation assets
2) Iterate over them and apply resignated flag to sender wallet